packages/amplify-e2e-core/src/utils/git-operations.ts

```diff
@@ -31,6 +31,10 @@ export const gitCleanFdX = async (cwd: string): Promise<void> => {
   await execa('git', ['clean', '-fdX'], { cwd });
 };
 
+export const gitSetLongPaths = async (cwd: string): Promise<void> => {
+  await execa('git', ['config', '--global', 'core.longpaths', 'true'], { cwd });
+};
```

Comment:
would it be better to add it here https://github.com/aws-amplify/amplify-cli/blob/2ceabaf826c9e86ace944dc1863b87b5dc1b297e/.circleci/config.base.yml#L462-L464 ?